content/docs/command-reference/exp/run.md

```diff
@@ -137,19 +142,20 @@ CPU cores).
   name will generated by default, such as `exp-f80g4` (based on the experiment's
   hash).
 
+- `--temp` - run this experiment in the background, and outside your workspace
+  (in `.dvc/tmp/exps`). Useful to continue working while a long experiment runs.
```

Comment:
My only concern is that "backgound" sounds a bit like `exp run &` which isn't exactly it. with `--temp/queue` you probably need a separate terminal for your "foreground" work. Should probably rephrase... Ideas?

Reply:
I think it would be better to just say "outside your workspace" instead of "in the background" like it is documented in `--run-all`.

Honestly I think we can just document `--temp` as a shortcut/equivalent to `--queue` + `--run-all` for single experiment runs.

Reply:
> I think it would be better to just say "outside your workspace"

👍 

> I think we can just document `--temp` as a shortcut/equivalent to `--queue` + `--run-all`

Not sure it makes sense to have to read two other options to understand `--temp`, and it's unclear whether `--temp` should refer back to `--queue`/`--run-all` or vice versa.

Simply removing "in the background" seems good enough to me:

```suggestion
- `--temp` - run this experiment outside your workspace
  (in `.dvc/tmp/exps`). Useful to continue working while a long experiment runs.
```

Reply:
> better to just say "outside your workspace"... Simply removing "in the background"

Done (7a1e5518).

> just document --temp as a shortcut/equivalent to --queue + --run-all for single experiment runs

Agree with @dberenbaum that it's best to doc `--temp` on its own as it has it's own uses. The fact that it relates closely with `--queue` is more of an implementation detail.

Reply:
> The fact that it relates closely with --queue is more of an implementation detail.

That said, it is explained in the Queued runs section, so the connection is reflected there.